Dart/src/com/jetbrains/lang/dart/ide/refactoring/ServerRefactoring.java

```diff
@@ -73,13 +73,16 @@ public void run(@NotNull ProgressIndicator indicator) {
     if (serverErrorStatus != null) {
       return serverErrorStatus;
     }
+    if (finalStatus == null) {
+      return null;
+    }
     RefactoringStatus result = new RefactoringStatus();
     result.merge(optionsStatus);
     result.merge(finalStatus);
     return result;
   }
 
-  @NotNull
+  @Nullable
   public RefactoringStatus checkInitialConditions() {
     ProgressManager.getInstance().run(new Task.Modal(null, "Checking initial conditions", true) {
```

Comment:
Cosmetics: messages like this give no clue to the user about what IDE is doing and what will fail if user clicks Stop. IMHO refactoring name would be better, for example "Inline Variable". May be even better message could be composed.